PXDConverter/MarshalService.cs

```diff
@@ -1,26 +1,33 @@
-﻿using System;
+using System;
 using System.Runtime.InteropServices;
 
 namespace PXDConverter
 {
+    interface PXDDecompressLibrary
```

Comment:
I would move all new interfaces/classes/enums to separate files.
That would make the code more readable when it grows

Reply:
I will delete this interface because it is useless already